### PR TITLE
publishConfig.directory docs

### PR DESCRIPTION
**Pull request by PabloSzx**

### CODE REVIEW

**Comment by zkochan** [comment]
it would probably make sense to add a section about it on this page: https://pnpm.io/package_json#publishconfig

`## publishConfig.directory`

and the changes should be duplicated in the docs directory as well.

**Comment by PabloSzx** [comment]
@zkochan I added the section on both, and assumed the version "6.7.0", but I don't know if you prefer to remove the publishConfig.directory reference from the cli docs and just use the "publishConfig.directory" section

File: docs/package_json.md [hunk 1]
```diff
@@ -108,6 +108,24 @@ Will be published as:
...
+You also can use the field `publishConfig.directory` to customize the published subdirectory relative to the current `package.json`.
+
+It is expected to have a modified version of the current package in the specified directory (usually using third party build tools).
+
+```json
+{
+  "name": "foo",
+  "version": "1.0.0",
+  "publishConfig": {
+    "directory": "dist"
```
Comment (zkochan) [inline docs/package_json.md:124]: maybe also metion that the dist folder should contain a package.json as well.

**Comment by zkochan** [comment]
I will merge it when 6.7 will be published

**Pull request merged**